packages/interledger/src/accounts/types/deposit.ts

```diff
@@ -1,18 +1,26 @@
-interface Deposit {
-  depositId?: bigint
+interface DepositOptions {
+  id?: string
   amount: bigint
 }
 
-export interface AccountDeposit extends Deposit {
+export interface AccountDeposit extends DepositOptions {
   accountId: string
 }
 
-export interface LiquidityDeposit extends Deposit {
+export interface LiquidityDeposit extends DepositOptions {
   assetCode: string
   assetScale: number
 }
 
+export type Deposit = Required<AccountDeposit> & {
+  // createdTime: bigint
+}
+
 export enum DepositError {
   DepositExists = 'DepositExists',
   UnknownAccount = 'UnknownAccount'
 }
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
+export const isDepositError = (o: any): o is DepositError =>
```

Comment:
Would `unknown` work better than `any` here?

Reply:
I get:
```
error TS2345: Argument of type 'unknown' is not assignable to parameter of type 'CreateAccountError'.
  Type 'unknown' is not assignable to type 'CreateAccountError.UnknownSuperAccount'.
```

which seems expected given the type of check happening:
https://2ality.com/2020/06/type-guards-assertion-functions-typescript.html#user-defined-type-guards
>Alas, the type `unknown` does not let us make the function call in line A. Therefore, we have to use the type `any` for the parameter `value`.